libs/bootstrap-desktop.sh

```diff
@@ -18,4 +18,33 @@ else
   if [ ! -d "${SQLCIPHER_LIB_DIR}" -o ! -d "${OPENSSL_DIR}" -o ! -d "${APPSERVICES_PLATFORM_DIR}/nss" ]; then
     pushd libs && ./build-all.sh desktop && popd
   fi;
+
+  # NSS system libs check.
+  pushd "$(mktemp -d)"
+  echo '
```

Comment:
and thus autotools-rs was born

Reply:
Heh I have to say I got inspired by what they do :)